app/controllers/admin/api/cms/sections_controller.rb

```diff
@@ -44,7 +44,7 @@ def destroy
       @section.destroy
       respond_with @section, location: admin_api_cms_sections_path(@section)
     else
-      render_error status: :method_not_allowed, text: "This section can't be deleted"
+      render_error "Builtin resources can't be deleted", status: :unprocessable_entity
```

Comment:
There is another one here @jlledom :wink:

Reply:
Another one done!